htgettoken.spec

```diff
@@ -76,6 +78,7 @@ rm -rf $RPM_BUILD_ROOT
 #   That used to be an additional optional meaning of the --vaultalias option,
 #   but urllib3 requires only one name to match.
 # - Add setuptools build infrastructure
+# - Use wheels to build/install Python package
```

Comment:
How about being a little more descriptive about the implications of this, something like `, resulting in making htgettoken.main` directly callable by python programs.

Reply:
Sorry for the delay, this is now done. Building with wheels doesn't impact the way that the package can be used, but it does slightly improve the available metadata and means that entry point scripts aren't built to rely upon `pkg_resources`.

Reply:
Oh, so it's the previous line "Add setuptools build infrastucture" that made it possible to invoke htgettoken.main from a python program?  If so, how about adding that fact onto that entry?

Reply:
@DrDaveD, sorry for the delay.

> Oh, so it's the previous line "Add setuptools build infrastucture" that made it possible to invoke htgettoken.main from a python program? If so, how about adding that fact onto that entry?

Not quite, the `htgettoken.main` is independent of those (sort of*). I have now added an extra spec changelog entry to describe the refactoring itself, I hope this is satisfactory.

<sub>*Building with wheels only has useful implications when an entry point is created, so refactoring the script into a module doesn't require using wheels (or vice-versa) but using wheels is only useful after this refactoring.</sub>